tests/Query/BuilderTest.php

```diff
@@ -749,49 +749,30 @@ public function testWhereIn(): void
         $this->assertSame(2, $conn->table($tableName)->whereIn('bytesTest', [new Bytes(chr(10)), new Bytes(chr(20))])->count());
     }
 
-    public function testPartitionedDml(): void
+    public function test_partitionedDml(): void
     {
-        if (getenv('SPANNER_EMULATOR_HOST')) {
-            $this->markTestSkipped('Cannot test PartitionedDml on emulator');
-        }
-
         $conn = $this->getDefaultConnection();
         $tableName = self::TABLE_NAME_TEST;
+        $insertCount = 100;
 
         $insertValues = [];
-        for ($i = 0; $i < 20001; $i++) {
+        for ($i = 0; $i < $insertCount; $i++) {
             $insertValues[] = $this->generateTestRow();
         }
+        $conn->table($tableName)->insert($insertValues);
 
-        collect($insertValues)->chunk(50)->each(function(Collection $chunk) use($conn, $tableName) {
-            $insertValues = array_values($chunk->all());
-            $conn->table($tableName)->insert($insertValues);
-        });
-
-        // normal DML should throw error since its over 20000 rows
```

Comment:
DML mutation limit has been raised to 80k and PHP now runs of out memory before it reaches that point so removing test to check the mutation limit.